kie-pmml-bom/pom.xml

```diff
@@ -0,0 +1,590 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@gitgabrio 
FYI, removing build-boostrap repo, kie-pmml-bom is here but as soon as we are ready to remove also the knowledge dependency this will be moved in drools.git repo like everything else

Reply:
:+1: